tests/test_course.py

```diff
@@ -308,6 +308,33 @@ def test_get_quiz_fail(self, m):
         with self.assertRaises(ResourceDoesNotExist):
             self.course.get_quiz(settings.INVALID_ID)
 
+    # get_quiz_overrides()
+    def test_get_quiz_overrides(self, m):
+        register_uris({"course": ["get_quiz_overrides"]}, m)
+
+        overrides = self.course.get_quiz_overrides()
+        override_list = list(overrides)
+
+        self.assertEqual(len(override_list), 2)
+        self.assertIsInstance(override_list[0], QuizAssignmentOverrideSet)
+        self.assertTrue(hasattr(override_list[0], "quiz_id"))
+        self.assertTrue(hasattr(override_list[0], "due_dates"))
+        self.assertTrue(hasattr(override_list[0], "all_dates"))
+
+        self.assertTrue("id" in override_list[0].due_dates[0])
+        self.assertTrue("due_at" in override_list[0].due_dates[0])
+        self.assertTrue("unlock_at" in override_list[0].due_dates[0])
+        self.assertTrue("lock_at" in override_list[0].due_dates[0])
+        self.assertTrue("title" in override_list[0].due_dates[0])
+        self.assertTrue("base" in override_list[0].due_dates[0])
+
+        self.assertTrue("id" in override_list[0].all_dates[0])
+        self.assertTrue("due_at" in override_list[0].all_dates[0])
+        self.assertTrue("unlock_at" in override_list[0].all_dates[0])
+        self.assertTrue("lock_at" in override_list[0].all_dates[0])
+        self.assertTrue("title" in override_list[0].all_dates[0])
+        self.assertTrue("base" in override_list[0].all_dates[0])
```

Comment:
Consider iterating over a list of the attributes (i.e. `['id', 'due_at', ... , 'base']`) and the `all` function to make this more compact